easybuild/easyconfigs/n/NvidiaApex/NvidiaApex-20200325-cuda-10.1.eb

```
@@ -0,0 +1,42 @@
+easyblock = 'PythonPackage'
+
+name = 'NvidiaApex'
+local_commit = '8fac3a7'
+version = '20200325'
+
+req_py_majver = 3
+req_py_minver = 7
+
+homepage = 'https://github.com/nvidia/apex'
+description = "A PyTorch Extension: Tools for easy mixed precision and distributed training in Pytorch"
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/NVIDIA/apex/archive/']
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'download_filename': '%s.tar.gz' % local_commit,
+}]
+checksums = ['5584ab0f5d395b063a64e15233ee576740db14b2aa9a6c289c512b7dc69b2d88']
+
+dependencies = [
+    ('PyTorch', '1.5.0-cuda-10.1'),
```

```suggestion
    ('PyTorch', '1.5.0'),
```
If you remove the versionsuffix from the PyTorch recipe you also change the name of the next file and the dependency here

---

hm... shouldn't the PyTorch config be explicit in case we want to support its cpu version, or another cudatoolkit (e.g. cuda-10.2)?